### PR TITLE
Lesson04

### PR DESCRIPTION
**Pull request by Vl92**

### CODE REVIEW

File: Lesson04/HW04.Birthday/ConsoleDatePicker.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        internal void Show()
+        {
+            throw new NotImplementedException();
+        }
+
+        internal DateTime GetDate()
+        {
+            throw new NotImplementedException();
+        }
+    }
```
Comment (Onix-one) [inline Lesson04/HW04.Birthday/ConsoleDatePicker.cs]: Have you used it somewhere? If not. Why is it in the program?

File: Lesson04/HW04.Birthday/Program.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Onix-one) [inline Lesson04/HW04.Birthday/Program.cs]: I have already talked. Need to remove unused usings

File: Lesson04/HW04.Birthday/Program.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        public int year_2, month_2, day_2;
+        public void info_t()
+        {
+            Console.Write("Год = ");
+            year_2 = int.Parse(Console.ReadLine());
+            Console.Write("Месяц = ");
+            month_2 = int.Parse(Console.ReadLine());
+            Console.Write("День = ");
+            day_2 = int.Parse(Console.ReadLine());
+        }
```
Comment (Onix-one) [inline Lesson04/HW04.Birthday/Program.cs]: Names must not contain underscores or numbers. Read about variable naming

File: Lesson04/HW04.Birthday/Program.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            {
+                result_month = month_2 + 12 - month;
+                result_year = year_2 - 1 - year;
+                Console.WriteLine("Получается {0} лет и {1} месяцов ", result_year, result_month);
+            }
+        }
+        static void Main()
+        {
+            Program vadym = new Program();
+            Console.WriteLine("Настоящая дата рождения : ");
```
Comment (Onix-one) [inline Lesson04/HW04.Birthday/Program.cs]: Why Vadim? Why not Alex or Ivan? )))) Need to change on var program = new Program (); in all places

File: Lesson04/HW04.Separator/ToolBarTray.cs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+﻿namespace HW04.Separator
+{
+    internal class ToolBarTray
+    {
+    }
+}
```
Comment (Onix-one) [inline Lesson04/HW04.Separator/ToolBarTray.cs]: What does this file do here? Remove

File: Lesson04/HW04.Combiner/Program.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+            Console.Write("Введите ваше имя: ");
+            string name = Console.ReadLine();
+            Console.Write("Введите вашу фамилию: ");
+            string surname = Console.ReadLine();
+
+            Console.Write("Здравствуйте, " + name +" " + surname +  ", сегодня хороший день");
+            
+
+
+            Console.ReadKey();
```
Comment (Onix-one) [inline Lesson04/HW04.Combiner/Program.cs]: Why so many spaces?  And usually use the words first name and last name.

File: Lesson04/HW04.SalaryBonusCalculator/Program.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            int experience5 = 25;
+            
+
+            Console.WriteLine("Введите вашу зарплату");
+            basic = Convert.ToString(Console.ReadLine());
+            Console.WriteLine("Ваша зарплата" + " " +basic);
+
+            Console.WriteLine("Введите ваш стаж работы");
+            experience = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Ваш стаж" + " " +experience);
```
Comment (Onix-one) [inline Lesson04/HW04.SalaryBonusCalculator/Program.cs]: Everything is fine, but what if the user enters "asdnkdnkvasdasdsczxcasddasd"

File: Lesson04/HW04.Birthday/Program.cs (hunk 2 above)
Comment (Vl92) [reply Lesson04/HW04.Birthday/Program.cs]: Done

File: Lesson04/HW04.Birthday/Program.cs (hunk 3 above)
Comment (Vl92) [reply Lesson04/HW04.Birthday/Program.cs]: Done

File: Lesson04/HW04.Birthday/ConsoleDatePicker.cs (hunk 1 above)
Comment (Vl92) [reply Lesson04/HW04.Birthday/ConsoleDatePicker.cs]: Done

File: Lesson04/HW04.Separator/ToolBarTray.cs (hunk 5 above)
Comment (Vl92) [reply Lesson04/HW04.Separator/ToolBarTray.cs]: Done

File: Lesson04/HW04.Combiner/Program.cs (hunk 6 above)
Comment (Vl92) [reply Lesson04/HW04.Combiner/Program.cs]: Done

File: Lesson04/HW04.Birthday/Program.cs (hunk 4 above)
Comment (Vl92) [reply Lesson04/HW04.Birthday/Program.cs]: Done

File: Lesson04/HW04.SalaryBonusCalculator/Program.cs (hunk 7 above)
Comment (Vl92) [reply Lesson04/HW04.SalaryBonusCalculator/Program.cs]: Done

**Pull request merged**